servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -235,7 +235,7 @@ RID RenderForwardMobile::RenderBufferDataForwardMobile::get_color_fbs(Framebuffe
 			RID render_target = render_buffers->get_render_target();
 			ERR_FAIL_COND_V(render_target.is_null(), RID());
 			RID target_buffer;
-			if (texture_storage->render_target_get_msaa(render_target) == RS::VIEWPORT_MSAA_DISABLED) {
+			if (view_count > 1 || texture_storage->render_target_get_msaa(render_target) == RS::VIEWPORT_MSAA_DISABLED) {
```

Comment:
We are just force disabling MSAA when using XR?

Reply:
Ah right, there is no point in supporting 2D MSAA at all if we are skipping 2D. Ideally we wouldn't render to the 2D render target at all in this case. We should be rendering directly to the swapchain.